client/post-editor/editor-drawer/index.jsx

```diff
@@ -240,7 +242,23 @@ const EditorDrawer = React.createClass( {
 					isNew={ this.props.isNew }
 				/>
 			</AccordionSection>
-		)
+		);
+	},
+
+	renderSeo: function() {
+		if ( ! config.isEnabled( 'manage/advanced-seo' ) || ! this.props.site ) {
```

Comment:
It might be nice to have something like this for all of the accordions, this isn't the only one that checks for the `site` before it renders. But it does get the data really quickly in production so it just looks like a page render jank.